docs/src/verification.md

```diff
@@ -1,602 +0,0 @@
----
-id: verification
-title: "Verification"
----
-
-:::caution
-We recommend [Web-First Assertions](./api/class-playwrightassertions) that automatically retry until the expected condition is met instead. This helps reducing the flakiness of the tests.
-:::
-
-<!-- TOC -->
-
-## Text content
-
-```js
-const content = await page.locator('nav:first-child').textContent();
```

Comment:
Not sure if the entire doc should deleted since the stuff after custom assertions is helpful. Was cleaning it up recently.

Reply:
Reverted this bit for now, will address separately.